samples/bluetooth/peripheral_lbs/sample.yaml

```diff
@@ -6,7 +6,7 @@ tests:
     build_only: true
     build_on_all: true
     platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf52dk_nrf52810
-      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuappns
+      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuappns thingy53_nrf5340_cpuapp
```

Comment:
Thanks for the change. Please fix commit message (you actually update peripheral_lbs sample :) )

Reply:
ops :p